### PR TITLE
[plugin.video.ustvnow] 2017.10.16

### PR DESCRIPTION
**Pull request by Lunatixz**
Please commit together.
https://github.com/xbmc/repo-scripts/pull/662
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.ustvnow/resources/lib/net.py [hunk 1]
```diff
@@ -252,16 +265,21 @@ def _fetch(self, url, form_data={}, headers={}, compression=True):
...
+                form_data = urllib.urlencode(form_data)
+                req = urllib2.Request(url, form_data)
+            req.add_header('User-Agent', self._user_agent)
+            for k, v in headers.items():
+                req.add_header(k, v)
+            if compression:
+                req.add_header('Accept-Encoding', 'gzip')
+            response = urllib2.urlopen(req)
+        except urllib2.HTTPError, e:
+            print e.code
```
Comment (learningit) [inline plugin.video.ustvnow/resources/lib/net.py]: Please use xbmc.log() instead of print.

File: plugin.video.ustvnow/resources/lib/net.py [hunk 2]
```diff
@@ -252,16 +265,21 @@ def _fetch(self, url, form_data={}, headers={}, compression=True):
...
+            if form_data:
+                form_data = urllib.urlencode(form_data)
+                req = urllib2.Request(url, form_data)
+            req.add_header('User-Agent', self._user_agent)
+            for k, v in headers.items():
+                req.add_header(k, v)
+            if compression:
+                req.add_header('Accept-Encoding', 'gzip')
+            response = urllib2.urlopen(req)
+        except urllib2.HTTPError, e:
```
Comment (Rechi) [inline plugin.video.ustvnow/resources/lib/net.py]: `except urllib2.HTTPError as e` for python3 compatibility (https://www.python.org/dev/peps/pep-3110/#compatibility)

File: plugin.video.ustvnow/resources/lib/net.py (hunk 1 above)
Comment (Lunatixz) [reply plugin.video.ustvnow/resources/lib/net.py]: Sorry, don't know how I missed that.

File: plugin.video.ustvnow/resources/lib/net.py (hunk 2 above)
Comment (Lunatixz) [reply plugin.video.ustvnow/resources/lib/net.py]: Thanks that document says either works? except_clause: 'except' [test [('as' | ',') test]]

Comment (Rechi) [reply plugin.video.ustvnow/resources/lib/net.py]: `except_clause: 'except' [test [('as' | ',') test]]` is what python 2.6 and python 2.7 accept
python3 only accepts `except_clause: 'except' [test ['as' NAME]]`

Comment (Lunatixz) [reply plugin.video.ustvnow/resources/lib/net.py]: ahhh, Duh :) thanks for the link, it will be helpful in the future.

**Comment by Lunatixz** [comment]
@learningit @Rechi made recommend changes, THX

**Comment by Lunatixz** [comment]
@learningit @Rechi can you merge this for me? its dependency was merged. THX

**Pull request merged**